### PR TITLE
dynamic contact details

### PR DESCRIPTION
**Pull request by harel**
Allows contact details to be dynamic from the db, and also overridden per market.  
Removes redundant models.

### CODE REVIEW

File: investment_report/models.py [hunk 1]
```diff
@@ -82,22 +81,33 @@ def __str__(self):
         return self.name
 
 
-class MarketLogo(models.Model):
```
Comment (depsiatwal) [inline investment_report/models.py:85]: do we need to think about migration of this data ?

File: investment_report/models.py [hunk 2]
```diff
@@ -82,22 +81,33 @@ def __str__(self):
...
-    name = models.CharField(max_length=255)
-    image = ImageField()
-    market = models.ForeignKey(Market)
-
-    def __str__(self):
-        return self.name
-
-
-class SectorLogo(models.Model):
```
Comment (depsiatwal) [inline investment_report/models.py:94]: do we need to think about migration of this data ?

**Review by depsiatwal — COMMENTED** [review]
LGTM 
Since 2 models are being removed is data being migrated and if so, I advice tests for the migration as well.

File: investment_report/models.py (hunk 1 above)
Comment (harel) [reply investment_report/models.py:85]: no - this is redundant and will be removed. 
The plan is to archive prod data before tihs is deployed. it's a new version of everything, including content and assets.

File: investment_report/models.py (hunk 2 above)
Comment (harel) [reply investment_report/models.py:94]: same as before. the data is removed.

**Review by depsiatwal — APPROVED** [review]
LGTM

**Pull request merged**